modules/core/test/image/testImageMorphology.cpp

```diff
@@ -100,10 +115,16 @@ TEST_CASE("Binary image morphology", "[image_morphology]")
 
       common_tools::imageErosionRef(I_morpho_ref, connexity);
       vpImageMorphology::erosion(I_morpho_tpl, (unsigned char)1, (unsigned char)0, connexity);
-      vpImageMorphology::erosion(I_morpho, connexity);
+      vpImageMorphology::erosion<unsigned char>(I_morpho, connexity);
 
       CHECK((I_morpho_ref == I_morpho_tpl));
       CHECK((I_morpho_ref == I_morpho));
+
+      #if defined(VISP_BUILD_DEPRECATED_FUNCTIONS)
+      vpImage<unsigned char> I_deprecated_erosion = I;
+      vpImageMorphology::erosion(I_deprecated_erosion, connexity);
+      CHECK((I_morpho_ref == I_deprecated_erosion));
+      #endif
```

Comment:
Same here

Reply:
Done in commit af0e02632. Thanks for the help =)